lms/djangoapps/support/static/support/jsx/program_enrollments/inspector.jsx

```diff
@@ -2,55 +2,174 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, InputText, StatusAlert, InputSelect } from '@edx/paragon';
 
-export const ProgramEnrollmentsInspectorPage = props => (
-  <form method="get">
-    {props.errors.map(errorItem => (
-      <StatusAlert
-        open
-        dismissible={false}
-        alertType="danger"
-        dialog={errorItem}
-      />
-    ))}
-    <div key="edX_accounts">
-      <InputText
-        name="edx_user"
-        label="edX account username or email"
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.username) || ''}
-      />
-    </div>
-    <div key="school_accounts">
-      <InputSelect
-        name="IdPSelect"
-        label="Learner Account Providers"
-        value="Select One"
-        options={
-          props.orgKeys
-        }
-      />
-
-      <InputText
-        name="external_user_key"
-        label="Institution user key from school. For example, GTPersonDirectoryId for GT students"
-        value={(props.learnerInfo && props.learnerInfo.user && props.learnerInfo.user.external_user_key) || ''}
-      />
+
+const renderUserSection = userObj => (
+  <div>
+    <h3>
+      edX account Info
+    </h3>
+    <div><span>Username: </span> {userObj.username}</div>
```

Comment:
I don't think that you need to put the key in a separate span element, unless you're planning to style it differently, as Kyle mentioned below. Otherwise, the following should work and would avoid an unnecessary element.

```
<div>Username: {userObj.username}</div>
```

Reply:
I want to style the label `username` differently for better visual display. So I will take Kyle and your feedback here to bold the label text